front/app/containers/IdeasShow/index.tsx

```diff
@@ -422,13 +416,7 @@ export class IdeasShow extends PureComponent<
                   <BodySectionTitle>
                     <FormattedMessage {...messages.proposedBudgetTitle} />
                   </BodySectionTitle>
-                  <StyledIdeaProposedBudget
-                    formattedBudget={getFormattedBudget(
```

Comment:
Used FormattedCurrency instead inside IdeaProposedBudget, so we only use 1 currency tool everywhere.